tests/integration/__init__.py

```diff
@@ -21,3 +21,7 @@ def workDir(self):
     @property
     def stdoutFile(self):
         return os.path.join(self.rootDir, self.getManifest()["stdoutFile"])
+
+    @property
+    def path(self):
+        return os.path.join(self.rootDir, self.getManifest()["path"])
```

Comment:
Where is this variable used?

Reply:
Ah, it's not. I originally used this in place of `.workDir` here:
https://github.com/Exabyte-io/express/blob/a8a9cd27433a52c6c764f4f61286910fc1924d55/tests/integration/parsers/apps/aiida/test_parser.py#L9
I'll remove this.